html2text/__init__.py

```diff
@@ -114,6 +114,7 @@ def __init__(self, out=None, baseurl='', bodywidth=config.BODY_WIDTH):
         self.pre = 0
         self.startpre = 0
         self.code = False
+        self.quote = False
```

Comment:
This option is not changeable by the CLI though, would be awesome if you could an option for this part so we could have it customized when user invoke via CLI

Reply:
@kurtmckee let me know once this is done so we can merge and cut a release.

Reply:
`self.quote` is only used as a flag to indicate the start and end of the quotation so that it would be possible to differentiate between start and stop quotation marks. Could you explain what CLI option you're wanting exposed?